SECP-service/src/main/resources/assets/app/views/password/change-password.html

```diff
@@ -0,0 +1,35 @@
+<div class="form-group content-area sample-show-hide">
+    <form name="frm" autocomplete="off" novalidate ng-submit="register()">
+        <h1 id="formHeader">Change Password</h1>
```

Comment:
should be on the left. Like other views. fa icon might be nice but not a necessity.

Reply:
you mean the text-alignment? And are you talking about adding the same fa icon from forgot-password or a new one?

Reply:
Nvm I got what you mean